islandora_solr_config/includes/admin.inc

```diff
@@ -31,6 +31,7 @@ function islandora_solr_config_admin_rss_settings($form, &$form_state) {
     'managingEditor' => '',
     'webMaster' => '',
   ));
+  $rss_limit = variable_get('islandora_solr_config_rss_limit', 50);
```

Comment:
You need to define this variable in the .install file, and get rid of it during uninstall.

Reply:
Latest commit addresses this.